src/main/java/org/folio/rest/impl/BLUsersAPI.java

```diff
@@ -731,24 +726,35 @@ public void deleteBlUsersByIdById(String id, List<String> include,
   }
 
   private boolean responseOk(Response r){
-    if(r != null && r.getBody() != null){
-      return true;
-    }
-    return false;
+    return r != null && r.getBody() != null;
   }
 
   private boolean isNull(Response r){
-    if(r != null){
-      return false;
-    }
-    return true;
+    return r == null;
   }
 
   private String getUsername(String token) {
     JsonObject payload = parseTokenPayload(token);
-    if(payload == null) { return null; }
-    String username = payload.getString("sub");
-    return username;
+    if (payload == null) {
+      return null;
+    }
+    return payload.getString("sub");
+  }
+
+  private String getUserId(String token) {
```

Comment:
nitpick: it would be nice to parse the token once and then get the components out of it.